### PR TITLE
Fix plural handling for zero quantity

### PR DESCRIPTION
**Pull request by tom-anders**
The removed code in the `mjs` file assumed that in all languages
`0` is followed by singular but that's not true e.g. for German and English.

I tested with French after removing the workaround and
updating localizations: `intl.Intl.pluralLogic` seems to handle
the plural logic correctly, choosing singular after 0 for French,
but plural for German and English.

Probably at the time that this code was added, `intl.Intl.pluralLogic`
was not used yet...?

### CODE REVIEW

**Comment by veloce** [comment]
I don't know about `pluralLogic`, I think it is actually due to me not understanding how pluralisation work in other languages.

Thanks for catching this.

File: scripts/update-arb-from-crowdin.mjs [hunk 1]
```diff
@@ -266,17 +266,17 @@ function transformTranslations(data, locale, module, makeTemplate = false) {
...
-    if (!plural.item.some((item) => item.$.quantity === 'zero')) {
-      const oneItem = plural.item.find((item) => item.$.quantity === 'one')
-      if (oneItem) {
-        plural.item.unshift({
-          $: { quantity: 'zero' },
-          _: oneItem._
-        })
-      }
-    }
+    //// add a zero quantity item if it doesn't exist
```
Comment (veloce) [inline scripts/update-arb-from-crowdin.mjs:269]: You can delete the block instead of commenting it out.

Comment (tom-anders) [reply scripts/update-arb-from-crowdin.mjs]: whoops, fixed

**Comment by tom-anders** [comment]
> I don't know about `pluralLogic`, I think it is actually due to me not understanding how pluralisation work in other languages.
> 
> Thanks for catching this.

Me neither until yesterday :D

**Comment by veloce** [comment]
@tom-anders can you re-run the script to solve the conflicts? thanks

**Comment by tom-anders** [comment]
> @tom-anders can you re-run the script to solve the conflicts? thanks

Done :+1:

**Comment by veloce** [comment]
Thanks!

**Pull request merged**